src/Tizen.NUI.Components/Controls/Extension/SlidingSwitchExtension.cs

```diff
@@ -36,16 +36,11 @@ public SlidingSwitchExtension() : base()
             slidingAnimation = new Animation(100);
         }
 
-        public override void OnClick(Button button, Button.ClickEventArgs eventArgs)
+        /// <inheritdoc/>
+        public override void OnSelectedChanged(Switch switchButton)
         {
-            if (button as Switch == null)
-            {
```

Comment:
looks like "throw new Exception(...)" has been removed. isn't it?
is it intentional removal?

Reply:
Yes :) Previous code, it was `OnClick(Button button)`. I moved body of `OnClick` to `OnSelectedChanged(Switch)`.
Unlike `OnSelectedChanged(Switch)`, the parameter type of `OnClick(Button button)` is Button so that it is not guaranteed the button is type of `Switch`. That was the reason why throwing exception was needed.

Reply:
OK. I see. Thank you for explanation~